shared_model/interfaces/iroha_internal/query_response_factory.hpp

```diff
@@ -44,17 +45,30 @@ namespace shared_model {
               assets,
           const crypto::Hash &query_hash) const = 0;
 
+      /**
+       * Create response for account asset query
+       * @param account_ids of assets to be inserted into the response
+       * @param asset_ids of assets to be inserted into the response
+       * @param balances of assets to be inserted into the response
+       * @param query_hash - hash of the query, for which response is created
+       * @return account asset response
+       */
+      virtual std::unique_ptr<QueryResponse> createAccountAssetResponse(
```

Comment:
Why haven't you removed the previous versions of functions? Those, which accept objects

Reply:
Because they are stil used in some tests